packages/vm/src/runTx.ts

```diff
@@ -588,7 +610,9 @@ async function _runTx(vm: VM, opts: RunTxOpts): Promise<RunTxResult> {
   })) as RunTxResult
 
   if (vm.common.isActivatedEIP(6800)) {
-    stateAccesses?.merge(txAccesses!)
+    stateAccesses?.merge(
+      txAccesses! as VerkleAccessWitnessInterface & BinaryTreeAccessWitnessInterface,
```

Comment:
Why do we need to type this as both here?

Reply:
I think the robots got confused on this one. Will adjust to `|`